data/net/minecraft/advancements/critereon/DeserializationContext.mapping

```diff
@@ -1,4 +1,7 @@
 CLASS net/minecraft/advancements/critereon/DeserializationContext
```

Comment:
Missing mappings for `deserializeConditions`.

Reply:
Yes